rs/execution_environment/tests/canister_logging.rs

```diff
@@ -25,10 +25,14 @@ const MAX_LOG_MESSAGE_LEN: usize = 4 * 1024;
 const TIME_STEP: Duration = Duration::from_nanos(111_111);
 
 // Change limits in order not to duplicate prod values.
+// Setting the instruction limit per round equal to the instruction limit per slice
+// means that the actual instruction limit used by the scheduler is 1
+// and thus only a single slice is executed in a round.
```

Comment:
> the actual instruction limit used by the scheduler is 1 and thus only a single slice is executed in a round.

this is what the behavior used to be before and thus I set updated the constants to match that given the new way of deriving the round instruction limit